src/st/class-wpml-pb-reuse-translations.php

```diff
@@ -68,10 +63,10 @@ private function find_existing_strings_for_new_strings( array $new_strings, arra
 	}
 
 	/**
-	 * @param int[] $new_strings
-	 * @param string[] $leftover_strings
+	 * @param int[]   $new_strings
+	 * @param array[] $leftover_strings
 	 *
-	 * @return array
+	 * @return array[]
```

Comment:
@strategio Scrutinizer here has a point: why you can't fix this inspection issue?

Reply:
Here's an example of the array @andreasciamanna:

```
array(
    'gf4544ds454sds542122sd' => array(
		'value' 	 => 'The string value',
		'context'    => 'the-string-context',
		'name'       => 'the-string-name',
		'id'         => 123,
		'package_id' => 123,
		'location'   => 123,
	),
)
```

I am not sure it will help to improve the PHPDoc, but I will add this structure as a comment in the main method.

Also, I already fixed this PHPDoc once after Scrutinizer was complaining, and now it comes with another suggestion.

Reply:
@strategio I thought it was just an array of strings (`string[]`). If it's more complicated, forget it :)